pkg/routeagent_driver/constants/constants.go

```diff
@@ -20,6 +20,12 @@ const (
 	SmGlobalnetEgressChain  = "SUBMARINER-GN-EGRESS"
 	SmGlobalnetMarkChain    = "SUBMARINER-GN-MARK"
 
+	// The following chains are added as part of GN 2.0 implementation
+	SmGlobalnetEgressChainForPods         = "SM-GN-EGRESS-PODS"
+	SmGlobalnetEgressChainHeadlessSvcPods = "SM-GN-EGRESS-HDLS-PODS"
```

Comment:
```suggestion
	SmGlobalnetEgressChainForHeadlessSvcPods = "SM-GN-EGRESS-HDLS-PODS"
```

Reply:
Just to be consistent in naming

Reply:
It was becoming tooo long. But anyways, I can change.